Quizzler-iOS13/Model/QuizManager.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QuizManager.swift
+//  Quizzler-iOS13
+//
+//  Created by Ronald Ngounou on 4/18/24.
+//  Copyright © 2024 The App Brewery. All rights reserved.
+//
+
+import Foundation
+
+class QuizManager{
+    
+    var questions: [ChoicesQuestions] = []
+    var questionNumber = 0
+    var score = 0
+    
+    init() {
+        let question1 = ChoicesQuestions(questionText: "What is the capital of France?", choices: ["London", "Paris", "Berlin"], correctAnswerIndex: 1, explanation: "Paris is the capital of France.")
+        
+        let question2 = ChoicesQuestions(questionText: "What is the capital of Belgium?", choices: ["London", "Paris", "Brussels"], correctAnswerIndex: 2, explanation: "Brussels is the capital of Belgium.")
```

Comment:
just a suggestion: move each parameter to a new line to make it easier to read (you can use `cmnd + M` shortcut for it)

Reply:
Thank you! ctrl  + M worked well on my side